boringtun/src/device/api.rs

```diff
@@ -155,7 +156,7 @@ impl Device {
 fn api_get(writer: &mut BufWriter<&UnixStream>, d: &Device) -> i32 {
     // get command requires an empty line, but there is no reason to be religious about it
     if let Some(ref k) = d.key_pair {
-        writeln!(writer, "private_key={}", encode_hex(k.0.as_bytes()));
```

Comment:
What are we changing here?

Reply:
Again, dumping a private key like this is a terrible idea.